demo/ansible-target-files/retreive_secrets_in_target.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+echo Conjur identity is:
```

Comment:
Can you quote your `echo`s? Makes it a lot easier to understand what is being echoed.